stripe2qbo/db/models.py

```diff
@@ -1,3 +1,4 @@
+from typing import Literal, Optional
```

Comment:
These I think are unused?

Did you do this locally? I'm surprised the pre-commit hook didn't catch this

Reply:
You might need to run `$ pre-commit install`